clients/linux/src/account.rs

```diff
@@ -11,22 +11,26 @@ use gtk::{
     EntryIconPosition as GtkEntryIconPosition, Grid as GtkGrid, Image as GtkImage,
     Label as GtkLabel, Menu as GtkMenu, MenuItem as GtkMenuItem, Paned as GtkPaned,
     ProgressBar as GtkProgressBar, ScrolledWindow as GtkScrolledWindow, Separator as GtkSeparator,
-    Spinner as GtkSpinner, Stack as GtkStack, WrapMode as GtkWrapMode,
+    Spinner as GtkSpinner, Stack as GtkStack, TextWindowType, WrapMode as GtkWrapMode,
 };
 use sourceview::prelude::*;
 use sourceview::View as GtkSourceView;
 use sourceview::{Buffer as GtkSourceViewBuffer, LanguageManager};
 
 use crate::backend::{LbCore, LbSyncMsg};
-use crate::closure;
 use crate::editmode::EditMode;
-use crate::error::LbResult;
+use crate::error::{LbErrTarget, LbError, LbResult};
 use crate::filetree::FileTree;
 use crate::messages::{Messenger, Msg, MsgFn};
 use crate::settings::Settings;
+use crate::util::gui::LEFT_CLICK;
 use crate::util::{gui as gui_util, gui::RIGHT_CLICK};
```

Comment:
Please give the imports a once over in the future. Right here, LEFT_CLICK can just be appended to the util import right below it.